aspnetcore/signalr/security.md

```diff
@@ -28,7 +28,17 @@ For more information on configuring CORS, see [Enable Cross-Origin Requests (COR
 
 * Allow the specific expected origins. Allowing any origin is possible but is **not** secure or recommended.
 * HTTP methods `GET` and `POST` must be allowed.
-* Credentials must be enabled, even when authentication is not used.
+* Credentials must be enabled:
+  * Even when authentication is not used.
+  * Because there is no customization in the browser client to not use credentials.
```

Comment:
I would not mention this at all, and it doesn't indicate the real reasons credentials are enabled.

Reply:
Yeah, this isn't correct. There **is** customization in the browser to do this (and we've even decided to expose that at some point moving forward.

I'd perhaps rephrase this whole bullet point (line 31-33) as something like (feel free to continue playing with style):

```
* Credentials must be allowed in order for cookie-based sticky sessions to work correctly. They must be enabled even when authentication is not used.
```